src/app/core-components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="popup-container {{ !isContainerVisible ? 'hide' : '' }}">
```

Comment:
Почему сделал именно так?
В Angular можно биндить классы через следующий синтаксис:
```suggestion
<div class="popup-container" [class.hide]="!isContainerVisible">
```
На мой взгляд это и гибче и читаемей выходит, что думаешь?

Reply:
Согласен, по привычке так сделал. Исправил.